### PR TITLE
Joyl 118

### PR DESCRIPTION
**Pull request by nguyennamthai**

### CODE REVIEW

File: dpx.yml [hunk 1]
```diff
@@ -16,10 +19,33 @@ services:
...
+      - target: 5432
+        published: 5433
+        protocol: tcp
+        mode: host
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=secret
+      - PGDATA=/var/lib/postgresql/data/pgdata
+    volumes:
+      - "./pgdata:/var/lib/postgresql/data/pgdata"
```
Comment (gzaro) [inline dpx.yml]: Are you sure that pgdata directory exists? Makefile must be updated in order to create this directory (make start) and clean it up (make distclean). Also maybe a clearer name, like rest-db would be better.

**Review by gzaro — CHANGES_REQUESTED** [review]

File: dpx.yml (hunk 1 above)
Comment (nguyennamthai) [reply dpx.yml]: Can you take another look? I haven't used makefile, so not sure if the changes are correct.

**Comment by gzaro** [comment]
if rest-db directory exists, 'make start' fails. Maybe mkdir -p will fix this. 'make distclean' fails because rest-db cannot be deleted.

File: dpx.yml (hunk 1 above)
Comment (nguyennamthai) [reply dpx.yml]: I have fixed the mkdir with -p option as you said. But I don't see any problem with distclean - it works fine. Can you check it again?

**Pull request closed without merging**